src/main/java/com/weekend/assignment/SumOfCharacters.java

```diff
@@ -0,0 +1,27 @@
+package com.weekend.assignment;
+
+public class SumOfCharacters {
+
+		static int findTheSum(String alpha)
+		{
+			int score = 0;
+			for(int i=0;i<alpha.length();i++)
+			{
+				if(alpha.charAt(i)>='A'&&alpha.charAt(i)<='Z')
+					score+=alpha.charAt(i)-'A'+1;
+				else
+					score+=alpha.charAt(i)-'a'+1;
+			}
+			return score;
+		}
+		public static void main(String[] args) {
+			String S = "abc";

```

Comment:
add		Scanner scanner = new Scanner(System.in);
		String s = scanner.nextLine();
		scanner.close();
		return s;

Reply:
i will check later